vault/data_source_kubernetes_auth_backend_role_test.go

```diff
@@ -98,7 +97,8 @@ func TestAccKubernetesAuthBackendRoleDataSource_full(t *testing.T) {
 		CheckDestroy: testAccCheckKubernetesAuthBackendConfigDestroy,
 		Steps: []resource.TestStep{
 			{
-				Config: testAccKubernetesAuthBackendRoleConfig_full(backend, role, ttl, maxTTL, audience),
+				Config: testAccKubernetesAuthBackendRoleDataSourceConfig_full(
+					backend, role, "serviceaccount_name", ttl, maxTTL, audience),
```

Comment:
Are we missing a check for `alias_name_source == "serviceaccount_name"` here?

Reply:
Good eye! Must have gotten dropped...

Reply:
Fixed in https://github.com/hashicorp/terraform-provider-vault/pull/1336/commits/a8b5fde9f877fe22d0d5492f74054573df6275e9